packages/web/components/Card/TechnologyCard.js

```diff
@@ -50,7 +50,7 @@ const TechnologyCard = ({ id, slug, title, costs, thumbnail, likes, users, type
 				</Link>
 				<TextContainer>
 					<TextPill>
-						{typesEnum.find((typeEnum) => typeEnum.value === type)?.label}
+						{typesEnum.find((typeEnum) => typeEnum.value === type)?.label || type}
```

Comment:
Take a look at prepareTechnology:

`technologyForAlgolia.type = TECHNOLOGY_TYPES[technologyData.type] || defaultTermMale;`

We don't need typesEnum anymore because `type` is already transformed when indexed to Algolia.

Maybe I'm talking shit, that's what I remember

Reply:
Sorry, bro. We still need this one because not all cards are being used with Algolia data